common/src/warp_runner/manager/commands/multipass_commands.rs

```diff
@@ -459,6 +400,26 @@ pub async fn handle_multipass_cmd(cmd: MultiPassCmd, warp: &mut super::super::Wa
     }
 }
 
+async fn update_identity_with_correct_values(
+    id: &mut Result<Identity, Error>,
```

Comment:
Maybe we can have this be `&mut Identity` instead and try the result early on and return any errors prior to setting the values into the wrapper